promql/parser/lex_test.go

```diff
@@ -180,6 +180,18 @@ var tests = []struct {
 			}, {
 				input:    "1y",
 				expected: []Item{{DURATION, 0, "1y"}},
+			}, {
+				input:    "1h6ms",
+				expected: []Item{{DURATION, 0, "1h6ms"}},
+			}, {
+				input:    "10m5s",
+				expected: []Item{{DURATION, 0, "10m5s"}},
+			}, {
+				input:    "10m5h",
```

Comment:
That this is lexed is an implementation detail, as it'll be rejected later on. I'd not test it here.

Reply:
ok